src/tests/ftest/control/dmg_pool_query_ranks.py

```diff
@@ -85,10 +85,12 @@ def test_pool_query_ranks_mgmt(self):
         self._verify_ranks(enabled_ranks, data, "enabled_ranks")
         self._verify_ranks(disabled_ranks, data, "disabled_ranks")
 
+        self.log_step(f"Waiting for rebuild to start after excluding pool rank {exclude_rank}")
+        self.pool.wait_for_rebuild_to_start()
```

Comment:
I thought @wangshilong intentionally removed this before?

Reply:
No, this is fine.